pages/common/jmap.md

```diff
@@ -1,6 +1,7 @@
 # jmap
 
 > Java Memory Map Tool.
+> More information: <https://docs.oracle.com/javase/7/docs/technotes/tools/share/jmap.html>.
```

Comment:
We prefer Manned in general, and I think it fits for this page:
```suggestion
> More information: <https://manned.org/jmap>.
```

Reply:
Both man pages contain the exact same content and I assume `jmap` was made by oracle, so I'd say we should keep the old one.

Reply:
Yeah, `jmap` is part of Java, so unfortunately the Oracle website is the preferred source here.